testdata/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.16 as builder
+WORKDIR /app
+COPY ./ ./
+
+RUN go mod download
```

Comment:
`go build` should get all the required deps automatically if they are not vendored